drivers/hmis/app/models/hmis/project_config.rb

```diff
@@ -5,6 +5,7 @@
 
 class Hmis::ProjectConfig < Hmis::HmisBase
   self.table_name = 'hmis_project_configs'
+  pc_t = Hmis::ProjectConfig.arel_table # todo @martha - is this ok?
```

Comment:
I think this is a little funny, I haven't seen us use this pattern. I would keep it as it was (re-declare in the other scope). You could also use `arel_table` directly (instead of `pct_t`, you should have access to that here since it's a class method on the class.